index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.0/css/bootstrap.min.css"
+    />
+    <title>Team-Profile-Generator</title>
+  </head>
+  <body>
+    <nav
+      class="
+        navbar navbar-nav navbar-center navbar-light
+        progress-bar-striped progress-bar-animated
+        bg-danger
+      "
+    >
+      <p class="mb-0 h1 text-white font-weight-bold mb-1 mt-1">My Team</p>
+    </nav>
+    <div class="container d-flex flex-wrap h-100"><div class="card mx-auto mt-5"" style="width: 18rem">
+        <div class="card-body bg-light">
+          <h2 class="card-title text-danger font-weight-medium">Bob</h5>
+          <h4 class="mb-3 text-primary">
+          Manager
+          </h6>
+          <ul class="list-group list-group-flush">
+            <li class="list-group-item">ID: 123 </li>
+            <li class="list-group-item">Email: info@info.com</li><li class="list-group-item">Office number: 1</li></ul>
```

Comment:
When a user clicks on the email, it should open up a new mail window. This can be done using an anchor tag with a mailto attribute, here's an example:
`
<a href="mailto:someone@example.com">Send email</a>
`